### PR TITLE
Fix issue 1236

### PR DESCRIPTION
**Pull request by EricMiddelhove**
fix issue #1236 
changed voice over according to comment from @saraolson 
my Habitica User-ID:

### CODE REVIEW

**Comment by saraolson** [comment]
thanks! i've added this to be reviewed and tested again. we'll let you know if we need anything else

**Comment by saraolson** [comment]
@EricMiddelhove testing this, its still only saying 'difficulty [x]' with no indication of whether its on or off. also not saying whether habit control is off, only on.

**Comment by EricMiddelhove** [comment]
@saraolson, It appears that the merging process did not went well. Some lines, I deleted in my commit are still in the develop branch. For some reasons, the files in this repository differ from the last commit I submitted for the pull request. Some line deletions I made were not applied. I commented on the important lines @phillipthelen

File: HabitRPG/UI/Tasks/TaskFormView.swift [hunk 1]
```diff
@@ -47,21 +65,10 @@ struct DifficultyPicker: View {
```
Comment (EricMiddelhove) [inline HabitRPG/UI/Tasks/TaskFormView.swift:65]: @phillipthelen The following line deletions are important

**Comment by saraolson** [comment]
@EricMiddelhove thanks for pointing that out! phillip will look into it happened and hopefully get it fixed up

**Comment by saraolson** [comment]
@EricMiddelhove thanks for your patience as we fixed everything up and reviewed again! everything looks good and i've added this contribution to your Habitica profile, as well as granted you a second tier :)

**Pull request merged**